--- origsrc/Remmina-1.0.0/remmina/CMakeLists.txt	2014-07-11 00:34:48.376845500 -0500
+++ src/Remmina-1.0.0/remmina/CMakeLists.txt	2014-07-11 00:56:26.321663500 -0500
@@ -92,6 +92,11 @@ add_executable(remmina ${REMMINA_SRCS})
 
 include_directories(${GTK_INCLUDE_DIRS})
 target_link_libraries(remmina ${GTK_LIBRARIES})
+set_target_properties(remmina PROPERTIES ENABLE_EXPORTS TRUE)
+
+if(CYGWIN)
+target_link_libraries(remmina intl)
+endif()
 
 find_package(X11)
 include_directories(${X11_INCLUDE_DIR})
--- origsrc/Remmina-1.0.0/remmina/remmina.pc.in	2012-02-10 12:54:23.000000000 -0600
+++ src/Remmina-1.0.0/remmina/remmina.pc.in	2014-07-11 01:08:26.531118500 -0500
@@ -7,6 +7,6 @@ Name: Remmina
 Description: The GTK+ Remote Desktop Client
 Version: @VERSION@
 Requires: 
-Libs: 
+Libs: -L${libdir} -lremmina
 Cflags: -I${includedir}
 
--- origsrc/Remmina-1.0.0/remmina-plugins/CMakeLists.txt	2014-07-11 00:34:50.596627400 -0500
+++ src/Remmina-1.0.0/remmina-plugins/CMakeLists.txt	2014-07-11 00:45:58.291913900 -0500
@@ -23,7 +23,7 @@ set(APPICON16_EMBLEMS_DIR "${REMMINA_DAT
 set(APPICON22_EMBLEMS_DIR "${REMMINA_DATADIR}/icons/hicolor/22x22/emblems")
 
 set(REMMINA_COMMON_INCLUDE_DIRS ${GTK_INCLUDE_DIRS})
-set(REMMINA_COMMON_LIBRARIES ${GTK_LIBRARIES})
+set(REMMINA_COMMON_LIBRARIES remmina ${GTK_LIBRARIES} ${X11_X11_LIB} intl)
 
 find_suggested_package(LIBSSH)
 if(LIBSSH_FOUND)
--- origsrc/Remmina-1.0.0/remmina-plugins-gnome/CMakeLists.txt	2012-02-10 12:54:23.000000000 -0600
+++ src/Remmina-1.0.0/remmina-plugins-gnome/CMakeLists.txt	2014-07-11 00:45:43.926589700 -0500
@@ -25,7 +25,7 @@ add_library(remmina-plugins-gnome ${REMM
 set_target_properties(remmina-plugins-gnome PROPERTIES PREFIX "")
 
 include_directories(${GTK_INCLUDE_DIRS})
-target_link_libraries(remmina-plugins-gnome ${GTK_LIBRARY_DIRS})
+target_link_libraries(remmina-plugins-gnome ${GTK_LIBRARIES})
 
 find_required_package(GNOMEKEYRING)
 if(GNOMEKEYRING_FOUND)
