--- origsrc/Remmina-1.0.0/CMakeLists.txt	2014-07-11 00:34:49.669009600 -0500
+++ src/Remmina-1.0.0/CMakeLists.txt	2014-07-11 00:54:25.070766600 -0500
@@ -107,4 +107,4 @@ endif()
 
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/config.h.in ${CMAKE_CURRENT_SOURCE_DIR}/config.h)
 
-install(DIRECTORY include/remmina DESTINATION include/remmina FILES_MATCHING PATTERN "*.h")
+install(DIRECTORY remmina/include/remmina DESTINATION include/remmina FILES_MATCHING PATTERN "*.h")
--- origsrc/Remmina-1.0.0/remmina/CMakeLists.txt	2014-07-11 00:34:48.376845500 -0500
+++ src/Remmina-1.0.0/remmina/CMakeLists.txt	2014-07-11 00:56:26.321663500 -0500
@@ -145,7 +150,9 @@ add_subdirectory(po)
 add_subdirectory(icons)
 add_subdirectory(desktop)
 
-install(TARGETS remmina DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(TARGETS remmina
+	RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+	ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/remmina.pc.in ${CMAKE_CURRENT_BINARY_DIR}/remmina.pc @ONLY)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/remmina.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
