--- origsrc/Remmina-1.2.0-rcgit.27/CMakeLists.txt	2018-02-14 16:33:54.000000000 -0600
+++ src/Remmina-1.2.0-rcgit.27/CMakeLists.txt	2018-02-25 15:46:17.346402800 -0600
@@ -60,7 +60,7 @@ endif()
 
 if(CMAKE_COMPILER_IS_GNUCC)
         set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
-        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
+        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${CMAKE_C_COMPILE_OPTIONS_PIC}")
         if(WITH_SSE2)
                 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -msse2")
         endif()
--- origsrc/Remmina-1.2.0-rcgit.27/remmina-plugins/rdp/rdp_event.c	2018-02-14 16:33:54.000000000 -0600
+++ src/Remmina-1.2.0-rcgit.27/remmina-plugins/rdp/rdp_event.c	2018-02-25 15:46:42.024133300 -0600
@@ -43,7 +43,6 @@
 #include <gdk/gdkkeysyms.h>
 #include <cairo/cairo-xlib.h>
 #include <freerdp/locale/keyboard.h>
-#include <execinfo.h>
 
 static void remmina_rdp_event_on_focus_in(GtkWidget* widget, GdkEventKey* event, RemminaProtocolWidget* gp)
 {
