--- origsrc/Remmina-1.2.0-rcgit.19/remmina/CMakeLists.txt	2017-07-26 02:34:51.689903800 -0500
+++ src/Remmina-1.2.0-rcgit.19/remmina/CMakeLists.txt	2017-07-26 02:35:11.426260900 -0500
@@ -194,7 +194,7 @@ add_subdirectory(desktop)
 add_subdirectory(external_tools)
 add_subdirectory(ui)
 
-install(TARGETS remmina DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(TARGETS remmina RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
 install(DIRECTORY include/remmina/
     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/remmina
     FILES_MATCHING PATTERN "*.h")
--- origsrc/Remmina-1.2.0-rcgit.19/plugins/nx/CMakeLists.txt	2017-07-24 04:11:37.000000000 -0500
+++ src/Remmina-1.2.0-rcgit.19/plugins/nx/CMakeLists.txt	2017-07-26 02:35:01.917451600 -0500
@@ -40,7 +40,7 @@ set(REMMINA_PLUGIN_NX_SRCS
         nx_plugin.h
         )
 
-add_library(remmina-plugin-nx ${REMMINA_PLUGIN_NX_SRCS})
+add_library(remmina-plugin-nx MODULE ${REMMINA_PLUGIN_NX_SRCS})
 set_target_properties(remmina-plugin-nx PROPERTIES PREFIX "")
 set_target_properties(remmina-plugin-nx PROPERTIES NO_SONAME 1)
 
--- origsrc/Remmina-1.2.0-rcgit.19/plugins/spice/CMakeLists.txt	2017-07-24 04:11:37.000000000 -0500
+++ src/Remmina-1.2.0-rcgit.19/plugins/spice/CMakeLists.txt	2017-07-26 02:35:01.924451100 -0500
@@ -36,7 +36,7 @@ set(REMMINA_PLUGIN_SPICE_SRCS
         spice_plugin_usb.c
         )
 
-add_library(remmina-plugin-spice ${REMMINA_PLUGIN_SPICE_SRCS})
+add_library(remmina-plugin-spice MODULE ${REMMINA_PLUGIN_SPICE_SRCS})
 set_target_properties(remmina-plugin-spice PROPERTIES PREFIX "")
 set_target_properties(remmina-plugin-spice PROPERTIES NO_SONAME 1)
 
